Implemented the AudioFeedback base class

suresh-gandhi opened the pull request:
Fixes #171.

**Changes**: 
* Implemented the AudioFeedback base class (belonging to the output.audio package) in Android version.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

suresh-gandhi committed:
Implemented the AudioFeedback base class

suresh-gandhi commented:
@CloudyPadmal @mariobehling Could you please review the PR. Let me know if something needs to be changed. Thanks.

ShridharGoel commented:
There are some unused fields and other issues as pointed out by Codacy. Please try to fix them.

jddeep commented on app/src/main/java/io/neurolab/main/output/audio/AudioFeedback.java:
```diff
@@ -0,0 +1,198 @@
+    private float[] defaultVolume = {.5f, .5f, .5f, .5f, .7f};
+    private String[] soundMixSamples = {"audio/pad_.wav", "audio/pad1.wav", "audio/lownoise.wav", "audio/pad2.wav", "audio/forest.wav"};
+    private Bitmap feedbackMixImage;
+    private int windowSize = 400;
```
I think these parameters are much related to the UI for the Feedback. No need of these in the base class. We can make UI related changes in the Activity and Fragment @suresh-gandhi .

jddeep commented on app/src/main/java/io/neurolab/main/output/audio/AudioFeedback.java:
```diff
@@ -0,0 +1,198 @@
+            currentMixPoint.set(x, y);
+        }
+
+        colorImage = BitmapFactory.decodeFile(ResourceManager.getInstance().getResource(context, "4colors.jpg").getAbsolutePath());
```
Same here. We can make UI related changes in the Activity/Fragment.

jddeep commented:
@suresh-gandhi Just go through the whole class once and keep only functionality related changes here in the base class. Because that will help us when we design and develop the UI. 🚀

suresh-gandhi commented:
@jddeep Thanks for your review. I removed android.graphics usages (to be reserved for UI Activity/Fragment class). However, volume, max volume and other variables are being loaded (their default values mostly) from the Config File via Resource Manager here and even getting modified at some places. Hence I have added getters for them to be directly accessed by the UI class to use them for display.

jddeep reviewed (approved):
@suresh-gandhi Good job 👍

suresh-gandhi commented:
@CloudyPadmal @mariobehling Could you please review and merge the PR.
